facade/src/main/java/org/jboss/pnc/facade/providers/ProductMilestoneProviderImpl.java

```diff
@@ -359,4 +386,187 @@ private List<Integer> getDependentMilestoneIds(CriteriaBuilder cb, String id) {
 
         return em.createQuery(buildQuery).getResultList();
     }
+
+    private Long getBuiltArtifactsInMilestone(CriteriaBuilder cb, String id) {
+        CriteriaQuery<Long> query = cb.createQuery(Long.class);
+
+        Root<Artifact> artifacts = query.from(Artifact.class);
+        Join<Artifact, BuildRecord> builtArtifacts = artifacts.join(Artifact_.buildRecord);
```

Comment:
This is very irritating.

Reply:
Glad to hear that, since the only goal of this variable was to irritate you.